depends/README.md

```diff
@@ -119,7 +119,7 @@ The following can be set when running make: `make FOO=bar`
 - `NO_BDB`: Don't download/build/cache BerkeleyDB
 - `NO_SQLITE`: Don't download/build/cache SQLite
 - `NO_USDT`: Don't download/build/cache packages needed for enabling USDT tracepoints
-- `MULTIPROCESS`: Build libmultiprocess (experimental)
+- `MULTIPROCESS`: Build libmultiprocess (experimental, no Windows and OpenBSD)
```

Comment:
In commit "ci: build multiprocess for all depends jobs" (d937f3abdccdfdf1008dceb7a4235b1cbab9a3ae)

This is a documentation change. not really a CI change, so ideally I think it wouldn't be part of this commit.

I think the most logical place to move this change would be commit 63dffe1795d9e7b1d3307e0982dbbea2f1563f0f in followup PR #31802, because that's the commit which adds the corresponding depends setting (`NO_MULTIPROCESS ?= $(if $(findstring mingw32,$(HOST))$(findstring openbsd,$(HOST)),1,)`)